gradle/lib/dependabot/gradle/update_checker/version_finder.rb

```diff
@@ -106,7 +106,7 @@ def filter_version_types(possible_versions)
           filtered = possible_versions.select { |v| matches_dependency_version_type?(v.fetch(:version)) }
           if possible_versions.count > filtered.count
             diff = possible_versions.count - filtered.count
-            Dependabot.logger.info("Filtered out #{diff} versions not matching the type")
+            Dependabot.logger.info("Filtered out #{diff} versions with a different classifier")
```

Comment:
In context with the other log messages around it, I think it would look like:

```
updater | Latest version is 1.0.0-android
updater | Requirements to unlock update_not_possible
updater | Requirements update strategy bump_versions
updater | Filtered out 156 versions with a different classifier
updater | No update possible for my-dependency 1.0.0-android
```

I think it's clear in context assuming folks working in the ecosystem know it's called a classifier, but we can add it if you think that would help.

Reply:
I know the folks in Maven ecosystem know, this is how it would be specified in the pom:

```xml
<dependency>
   <groupId>org.example</groupId>
   <artifactId>stuff</artifactId>
   <version>1.0.0</version>       
   <classifier>android</classifier>
</dependency>
```

Reply:
My 2c: the more verbose we are in logging; the more useful it is when debugging.

Reply:
I added the classifier name that we're filtering on to the log message.

Reply:
Yeah I think this is good, thanks! Tbh I am not sure if everyone involved knows what the classifier is, tbh, I didn't really know what it was either.